packages/coinstac-ui/app/render/components/consortium-single.js

```diff
@@ -140,18 +140,10 @@ class ConsortiumSingle extends Component {
 
             if (type === 'number') {
               fieldProps.onChange = event => {
-                if (typeof event === 'number') {
```

Comment:
`event` is no longer a number. It's always shaped like a DOM event, regardless of whether it is an actual event or an increment/decrement click.